woocommerce/woocommerce-gutenberg-products-block
# Add readme file to extend-rest-api directory and move internal doc into internal folder

## What

Adds a README and table of contents to the https://github.com/woocommerce/woocommerce-blocks/tree/trunk/docs/third-party-developers/extensibility/rest-api directory - it's bare bones but it makes the documentation a little easier to navigate.

It also moves the `Adding an endpoint to ExtendSchema` document to the internal developers directory.

Fixes #10574

## Why

> It also moves the `Adding an endpoint to ExtendSchema` document to the internal developers directory.

This is done because we don't expect third-party developers to add their own endpoints. That document was designed for internal consumption, so it makes more sense for it to live in the area for internal documentation.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Read the documentation. Check for typos or grammatical errors.
2. Check the links and ensure the point to the correct resource.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Skipping
